assets/src/components/dummyDetourPage.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from "react"
+import { fetchShapeForRoute } from "../api"
+import { DetourMap } from "./detours/detourMap"
+import { Shape } from "../schedule"
+
+export const DummyDetourPage = () => {
+  const [routeShape, setRouteShape] = useState<Shape | null>(null)
+
+  useEffect(() => {
+    fetchShapeForRoute("39").then((shapes) => {
+      setRouteShape(shapes[0])
+    })
+  }, [])
+
+  return (
+    <div className="l-page">
+      {routeShape && <DetourMap shape={routeShape} />}
```

Comment:
question: is there a reason for specifically using `DetourMap` here and not `DiversionPage`?

Reply:
Oh this is probably just left over from the version of the code that I had in my branch when I was solely working on my own `<DetourMap/>` component.

Reply:
Oh okay, I'm fine with it as is if it's changing incrementally in another branch.